CHANGELOG.md

```diff
@@ -184,6 +184,10 @@ Note: Can be used with `oxsecurity/megalinter@beta` in your GitHub Action mega-l
   - [cfn-lint](https://github.com/aws-cloudformation/cfn-lint) from 1.15.1 to **1.15.2** on 2024-09-30
 <!-- linter-versions-end -->
 
+## [v8.0.1] - 2024-08-19
+
+- Support shell scripts with no extension and only support perl shebangs at the beginning of a file in <https://github.com/oxsecurity/megalinter/pull/4076>
+
```

Comment:
Can you place that entry in the section that starts on line 7?

Reply:
Thanks, done